app/views/organisation_reports/without_users_index.html.erb

```diff
@@ -1,3 +1,7 @@
+<p><b> The Following email will be sent when a user is invited: </b></p>
+<p>
+  <%= render template: 'custom_devise_mailer/invitation_instructions', :locals => { :@resource => User.new } %>
```

Comment:
ah, as we discussed in the standup we need the @ as we are having to work with the mail template which requires them ...